TodoList.Web/Models/HomeViewModel.cs

```diff
@@ -7,5 +7,7 @@ public class HomeViewModel
     {
         public IEnumerable<TodoItem> RecentlyAddedTodos { get; set; }
         public IEnumerable<TodoItem> CloseDueToTodos { get; set; }
+        public IEnumerable<TodoItem> MonthlyToTodos { get; set; }
+        public CalendarViewModel CalendarViewModel { get; set; }
```

Comment:
Why those two properties are different? Couldn't `CalendarViewModel` contain the todo list items for its day?

Reply:
There is not currently a strong functional connection between the two, but I'll move MonthlyTodos into CalendarViewModel in anticipation of a future additions tying them together.